lib/morse_translator.rb

```diff
@@ -38,6 +38,41 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+    @morse_string = ""
+
+  end
+
+
+  def compare_units(element, letter)
+    if element == letter
+      @morse_string += @dictionary[letter]
+    end
+  end
+
+
+  def process_units(array_in)
+    letters = ('a'..'z').to_a
+    [1,2,3,4,5,6,7,8,9,0].to_a.each do |number|
+      letters << number.to_s
+    end
+    letters << " "
+    array_in.each do |element|
+      letters.each do |letter|
+        compare_units(element,letter)
+      end
+    end
+    @morse_string
+  end
+
+  def translate(string_in)
```

Comment:
I like that you have turned this into a CEO method - it really shows me that you understand return values and how methods can interact with each other!